docs/apidiffs/current_vs_latest/opentelemetry-sdk-extension-autoconfigure-spi.txt

```diff
@@ -2,3 +2,6 @@ Comparing source compatibility of  against
 ***! MODIFIED INTERFACE: PUBLIC ABSTRACT io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizer  (not serializable)
 	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
 	+++! NEW METHOD: PUBLIC(+) io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizer addTracerProviderCustomizer(java.util.function.BiFunction)
+===  UNCHANGED INTERFACE: PUBLIC ABSTRACT io.opentelemetry.sdk.autoconfigure.spi.traces.SdkTracerProviderConfigurer  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	+++  NEW ANNOTATION: java.lang.Deprecated
```

Comment:
was this missed in an earlier PR?

Reply:
Yes, I must not have run build locally in #4136